lib/job/schedule.ts

```diff
@@ -1,17 +1,15 @@
-'use strict';
-const date = require('date.js');
+// @ts-expect-error
```

Comment:
Nit; usually helpful to add extra comment to elaborate why error/warning was suppressed, even if it seems obvious.

Reply:
These aren’t meant to be kept. They should all be removed in the next PR we open for this.

Reply:
it is even the wrong comment operator. @ts-expect-error was designed to be used for unit tests, where you actually have type errors because you e.g. give incomplete objects violating the typings. This has to be actually an @ts-ignore

Reply:
No it doesn’t. Expect error can be used when you have code that’s correctly throwing a type issue and you expect it to stay that way for now. If we change any other code which solves this then this will error and the dev will know to remove it.